recipe/meta.yaml

```diff
@@ -41,8 +41,8 @@ requirements:
     - pycosat >=0.6.3
     - pyopenssl >=16.2.0
     - requests >=2.12.4,<3
-    - ruamel_yaml >=0.11.14,<0.15.55  # [py>27]
-    - ruamel_yaml >=0.11.14,<0.15.39|>=0.15.43,<0.15.55  # [py<=27]
+    - ruamel_yaml >=0.11.14  # [py>27]
+    - ruamel_yaml >=0.11.14,<0.15.39|>=0.15.43  # [py<=27]
```

Comment:
`conda 4.5.11` is compatible with `ruamel_yaml >=0.15.55`, so we can remove that restriction again.

Reply:
Ok, that can be in a future PR, unless you want to do it here

Reply:
Yeah, I think we can and should do it here, because `conda 4.5.11` is fine with those.

Reply:
@mbargull - do you have permissions to push to bot's branch?

Reply:
Yes, already did -- I added that comment to my changes/constraint removal afterwards :wink:

Reply:
This is good to go from my side!